src/smart-components/request/request-detail/request.js

```diff
@@ -39,7 +39,7 @@ export const Request = ({ item, isExpanded, toggleExpand }) => {
   };
 
   const checkCapability = (item, capability) => {
-    if (isApprovalAdmin(userPersona)) {
+    if (isApprovalAdmin(userRoles)) {
```

Comment:
^^

Reply:
These functions seem to me like good candidates for new hook aliases like `useIsApprovalAdmin` and others.